src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -773,12 +781,12 @@ public InmateDetail courtTransferIn(final String offenderNo, final RequestForCou
         offenderBooking.setInOutStatus(IN.name());
         offenderBooking.setLivingUnitMv(null);
 
-        deactivatePreviousMovements(offenderBooking);
 
         final MovementReason movementReason = getMovementReason(requestForCourtTransferIn.getMovementReasonCode(), latestExternalMovement.getMovementReason());
 
         offenderBooking.setStatusReason(MovementType.CRT.getCode() + "-" + movementReason.getCode());
         final LocalDateTime movementTime = getAndCheckMovementTime(requestForCourtTransferIn.getDateTime(), offenderBooking.getBookingId());
+        deactivatePreviousMovements(offenderBooking);
 
```

Comment:
this was done in the wrong order so validation never failed when a court return was done before a court out (which I suspect is never done in incorrectly in real world anyway)